index.js

```diff
@@ -0,0 +1,7 @@
+import './components/button.css'
+import './components/color.css'
+import './components/container.css'
+import './components/line.css'
+import './components/list.css'
+import './components/spacing.css'
+import './components/typography.css'
```

Comment:
* parcel supports doing css imports in js files, it's pretty neat
* since this file imports all the css files, they will be included in the dist.